src/test/java/domain/CarTest.java

```diff
@@ -0,0 +1,38 @@
+package domain;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+class CarTest {
+
+    @Test
+    @DisplayName("Car 생성 테스트")
+    void checkConstructor() {
+
+        String expected = "test";
+        Car car = new Car(expected);
+
+        String result = car.getName();
+        assertThat(result).isEqualTo(expected);
+    }
+
+    @Test
+    @DisplayName("Car 이름이 5글자 넘으면 예외발생")
+    void checkFailTest() {
+        assertThatThrownBy(() -> new Car("testTest"))
+                .isInstanceOf(IllegalArgumentException.class);
```

Comment:
```suggestion
        assertThatThrownBy(() -> new Car("testTest"))
                .isInstanceOf(IllegalArgumentException.class)
                .에러 메세지에 대한 검증
                ;
```
같은 IllegalArgumentException 이라도 원하는 결과값과는 다른 메세지를 가진 예외일 수 있습니다.
확실히 구분할 수 있도록 메세지까지 검증해보면 어떨까요??

Reply:
전혀 생각하지 못한 부분이었는데 감사합니다!